### PR TITLE
chore(api): add swagger desc for Arthas & Metric & Cypher & White API

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
## Purpose of the PR

dd swagger belong for arthas API

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-api/src/main/java/org/apache/hugegraph/api/arthas/ArthasAPI.java [hunk 1]
```diff
@@ -27,6 +27,8 @@
 import com.codahale.metrics.annotation.Timed;
 import com.taobao.arthas.agent.attach.ArthasAgent;
 
+import io.swagger.annotations.ApiOperation;
+import io.swagger.v3.oas.annotations.tags.Tag;
```
Comment (imbajin) [inline hugegraph-api/src/main/java/org/apache/hugegraph/api/arthas/ArthasAPI.java:31]: format import

File: hugegraph-api/src/main/java/org/apache/hugegraph/api/arthas/ArthasAPI.java [hunk 2]
```diff
@@ -35,6 +37,7 @@
 
 @Path("arthas")
 @Singleton
+@Tag(name = "ArthasAPI")
```
Comment (imbajin) [inline hugegraph-api/src/main/java/org/apache/hugegraph/api/arthas/ArthasAPI.java:40]: could also check other API like **metrics/cypher** if they also missed swagger? And show the current effect pic

Comment (SunnyBoy-WYH) [reply hugegraph-api/src/main/java/org/apache/hugegraph/api/arthas/ArthasAPI.java:40]: image
yes，i will update other

**Review by javeme — APPROVED** [review]
LGTM

**Comment by SunnyBoy-WYH** [comment]
image

**Comment by simon824** [comment]
Hi @SunnyBoy-WYH , when we update dependency version, we need to execute [./hugegraph-dist/scripts/dependency/regenerate_known_dependencies.sh](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-dist/scripts/dependency/regenerate_known_dependencies.sh) locally to update the dependency list [known-dependencies.txt](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-dist/scripts/dependency/known-dependencies.txt)

**Comment by SunnyBoy-WYH** [comment]
> Hi @SunnyBoy-WYH , when we update dependency version, we need to execute [./hugegraph-dist/scripts/dependency/regenerate_known_dependencies.sh](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-dist/scripts/dependency/regenerate_known_dependencies.sh) locally to update the dependency list [known-dependencies.txt](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-dist/scripts/dependency/known-dependencies.txt)

DONE,thanks for your notice~

**Review by javeme — APPROVED** [review]
LGTM

**Review by imbajin — APPROVED** [review]

**Pull request merged**